### PR TITLE
feat: add quickorder functionality as configurable feature

### PR DESCRIPTION
**Pull request by franciscomdrito**
## PR Type

 [x] Feature  

## What Is the New Behavior?

new quickorder feature available via feature toggle `quickorder`

## Does this PR Introduce a Breaking Change?

 [ ] Yes  
 [x] No

### CODE REVIEW

File: src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+@Component({
+  selector: 'ish-quickorder-page',
+  templateUrl: './quickorder-page.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class QuickorderPageComponent implements OnInit, OnDestroy {
+  numberOfRows = 5;
+  productsFromCsv: QuickOrderLine[] = [];
+  searchSuggestions = new Array<{ imgPath: string; sku: string; name: string }>();
+  csvFileStatus = CsvStatusMessages;
```
[the remark this answers is not in the document]
Comment (franciscomdrito) [reply src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts:20]: Was using it to have it available on the html:
  get csvStatusMessages() {
    return CsvStatusMessages;
  }
I'm using like this instead now, is that ok?

File: src/app/extensions/quickorder/models/quickorder-line.model.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export class QuickOrderLine {
```
[the remark this answers is not in the document]
Comment (franciscomdrito) [reply src/app/extensions/quickorder/models/quickorder-line.model.ts:1]: just recently learned that interfaces are no compiled, so less javascript :D

File: src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  status: number;
+
+  quickOrderForm: FormGroup;
+  csvForm: FormGroup;
+
+  private destroy$ = new Subject();
```
[the remark this answers is not in the document]
Comment (franciscomdrito) [reply src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts:26]: you are right. one of the things I need do some reading xD

File: src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    element = fixture.nativeElement;
+    store$ = TestBed.get(Store);
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+    expect(element).toBeTruthy();
+    expect(store$).toBeTruthy();
+    expect(() => fixture.detectChanges()).not.toThrow();
+    expect(element).toMatchSnapshot();
```
[the remark this answers is not in the document]
Comment (franciscomdrito) [reply src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.spec.ts:39]: I really need to invest some time on how to do proper tests xD

File: src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhhyi) [reply src/app/extensions/quickorder/pages/quickorder/quickorder-page.component.ts:20]: Oh, seems I missed the usage on the template. Yes, with getter would be better 😉

**Pull request merged**